README.md

```diff
@@ -3,3 +3,4 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 Its me Mario!
+Hello Mario!
```

Comment:
This is not the best option. Please remove

Reply:
removed as requested